opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchDataType.java

```diff
@@ -248,7 +292,7 @@ protected OpenSearchDataType cloneEmpty() {
   /**
    * Flattens mapping tree into a single layer list of objects (pairs of name-types actually),
    * which don't have nested types.
-   * See {@link OpenSearchDataTypeTest#traverseAndFlatten() test} for example.
+   * See {@see OpenSearchDataTypeTest#traverseAndFlatten() test} for example.
```

Comment:
I don't know if this javadoc is correct, I imagine the intended purpose was to link something but the rendered version includes all of the text:
```
Flattens mapping tree into a single layer list of objects (pairs of name-types actually), which don't have nested types. See @see OpenSearchDataTypeTest#traverseAndFlatten() test for example.
```

Reply:
Fixed in b04a92e1f.